examples/features/example_good_descriptor.cpp

```diff
@@ -0,0 +1,178 @@
+/*
+ * Software License Agreement (BSD License)
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2010-2012, Willow Garage, Inc.
```

Comment:
Unless you developed this code back in 2010-2012 while working at Willow Garage, please remove this copyright.
Instead, please add `Copyright (c) 2017-, Open Perception, Inc.`